test/format/format.sh

```diff
@@ -383,6 +384,22 @@ report_failure()
 	echo "$name: failure status reported" > $dir/$status
 }
 
+# Report all running CONFIGs
+report_running_configs()
+{
+	msg "Out of disk space detected, outputting non-failed CONFIGs"
```

Comment:
This message should be highly noticeable, for example,
```
	echo "############################################"
	echo "Out of disk space detected, outputting non-failed CONFIGs"
	echo "############################################"
```

Reply:
Done